src/CSSList/CSSList.php

```diff
@@ -296,6 +296,22 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before its sibling.
```

Comment:
In general, please always use the third person for the first sentence of a method documentation comment: "(This method) Does …":

```suggestion
     * Inserts an item before its sibling.
```

Reply:
Fixed.